lib/jdk.js

```diff
@@ -206,7 +206,7 @@ exports.detect = function detect(config, opts, finished) {
 							)
 							+ __('The __JAVA_HOME__ environment variable must point to the JDK and not the JRE (Java Runtime Environment).') + '\n'
 							+ __('You may want to reinstall the JDK by downloading it from %s.',
-								'__http://appcelerator.com/jdk__')
+								'__http://www.oracle.com/technetwork/java/javase/downloads/index.html__')
```

Comment:
Is there a reason not to use HTTPS on the URL?

Reply:
Fixed!